src/test/java/org/osiam/client/MeScopeIT.java

```diff
@@ -0,0 +1,430 @@
+public class MeScopeIT {
+
+    private static final String OWN_USER_ID = "cef9452e-00a9-4cec-a086-d171374ffbef";
+    private static final String OTHER_USER_ID = "834b410a-943b-4c80-817a-4465aed037bc";
+    private static final String GROUP_ID = "69e1a5dc-89be-4343-976c-b5541af249f4";
+
+    private final OsiamConnector oConnector = new OsiamConnector.Builder()
+            .setAuthServerEndpoint("http://localhost:8180/osiam-auth-server")
+            .setResourceServerEndpoint("http://localhost:8180/osiam-resource-server")
+            .setClientId("example-client")
+            .setClientSecret("secret")
+            .build();
+    private final Client client = ClientBuilder.newClient();
+
+    @Test
+    public void can_get_own_user() {
+        AccessToken accessToken = oConnector.retrieveAccessToken("marissa", "koala", Scope.ME);
+
+        User user = oConnector.getUser(OWN_USER_ID, accessToken);
+
+        assertThat(user.getUserName(), is(equalTo("marissa")));
+    }
+
+    @Test
+    public void can_update_own_user() {
+        AccessToken accessToken = oConnector.retrieveAccessToken("marissa", "koala", Scope.ME);
+        Email email = new Email.Builder()
+                .setValue("marrisa@example.com")
+                .setType(Email.Type.HOME)
+                .build();
+        UpdateUser updateUser = new UpdateUser.Builder()
+                .updateDisplayName("Marissa")
+                .addEmail(email)
+                .build();
+
+        User user = oConnector.updateUser(OWN_USER_ID, updateUser, accessToken);
+
+        assertThat(user.getDisplayName(), is(equalTo("Marissa")));
+        assertThat(user.getEmails().get(0).getValue(), is(equalTo("marrisa@example.com")));
+        assertThat(user.getEmails().get(0).getType(), is(equalTo(Email.Type.HOME)));
+    }
+
+    @Test
+    public void can_replace_own_user() {
+        AccessToken accessToken = oConnector.retrieveAccessToken("marissa", "koala", Scope.ME);
+        User originalUser = oConnector.getUser(OWN_USER_ID, accessToken);
+        Email email = new Email.Builder()
+                .setValue("marrisa@example.com")
+                .setType(Email.Type.HOME)
+                .build();
+        User replaceUser = new User.Builder(originalUser)
+                .setDisplayName("Marissa")
+                .addEmail(email)
+                .build();
+
+        User user = oConnector.replaceUser(OWN_USER_ID, replaceUser, accessToken);
+
+        assertThat(user.getDisplayName(), is(equalTo("Marissa")));
+        assertThat(user.getEmails().get(0).getValue(), is(equalTo("marrisa@example.com")));
+        assertThat(user.getEmails().get(0).getType(), is(equalTo(Email.Type.HOME)));
+    }
+
+    @Test
+    public void can_delete_own_user() {
+        AccessToken accessToken = oConnector.retrieveAccessToken("marissa", "koala", Scope.ME);
+
+        oConnector.deleteUser(OWN_USER_ID, accessToken);
```

Comment:
If no exception thrown, the user was deleted right? No need for searching the user again?